packages/synthchron-tauri/src/components/NewProjectModal.tsx

```diff
@@ -157,6 +157,7 @@ const NewProjectModal: React.FC<NewProjectModalProps> = ({
       projectModel: model, // TODO
       created: new Date().toJSON(),
       lastEdited: new Date().toJSON(),
+      lastOpened: (redirect ? new Date() : new Date(0)).toJSON(),
```

Comment:
is this way better than some null value?

Reply:
Yeah, cause "new Date()" gets the current time (which is 100% in session), and "new Date(0)" gets 1970, to make sure that the project will definitely not be considered part of the current session